### PR TITLE
Move TEC tests to tric, take 2

### PR DESCRIPTION
**Pull request by lucatume**
This PR objective is to move The Events Calendar tests to `tric`.

### CODE REVIEW

File: src/Tribe/Customizer/Global_Elements.php [hunk 1]
```diff
@@ -82,6 +82,32 @@ public function get_css_template( $template ) {
 				}
 			';
 
+			// Overrides for common components/full/buttons/_border.pcss.
+			$template .= '
+				.tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt),
+				.tribe-common a.tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt) {
+					border-color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:89]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: e36960144e1dd221e8213538c6f61833</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 2]
```diff
@@ -82,6 +82,32 @@ public function get_css_template( $template ) {
...
+					color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:90]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: 27801bc49daee7a89f2249e0cccd677c</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 3]
```diff
@@ -82,6 +82,32 @@ public function get_css_template( $template ) {
...
+				}
+			';
+
+			$template .= '
+				.tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):focus,
+				.tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):hover,
+				.tribe-common a.tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):focus,
+				.tribe-common a.tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):hover {
+					color: ' . $color_background . ';
+					background-color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:100]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: 59bc921c576274a8d2c149da22e57c0f</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 4]
```diff
@@ -82,6 +82,32 @@ public function get_css_template( $template ) {
...
+				}
+			';
+
+			$template .= '
+				.tribe-theme-twentyseventeen .tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):focus,
+				.tribe-theme-twentyseventeen .tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):hover,
+					background-color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:107]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: b3be6a05eb219e27a4596c24386cd15f</sub>

File: src/Tribe/Integrations/WPML/Option.php [hunk 5]
```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Handles the translation of the plugin settings.
+ *
+ * @since   TBD
+ *
+ * @package Tribe__Events__Integrations__WPML__Option
+ */
+
```
Comment (tr1b0t) [inline src/Tribe/Integrations/WPML/Option.php:9]: :no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: 4f78b643acdf15d4eddac11429e4d988</sub>

File: src/Tribe/Integrations/WPML/Option.php [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+/**
+ * Class Tribe__Events__Integrations__WPML__Option
+ *
+ * @since TBD
+ */
+class Tribe__Events__Integrations__WPML__Option {
+
```
Comment (tr1b0t) [inline src/Tribe/Integrations/WPML/Option.php:17]: :no_entry_sign: Class name is not valid; consider Tribe_Events_Integrations_WPML_Option instead

<sub>hash: 4aca20c4324773aedc3206e812d317dd</sub>

File: src/Tribe/Main.php [hunk 7]
```diff
@@ -35,7 +35,7 @@ class Tribe__Events__Main {
 		const VENUE_POST_TYPE     = 'tribe_venue';
 		const ORGANIZER_POST_TYPE = 'tribe_organizer';
 
-		const VERSION             = '5.1.6';
+		const VERSION             = '5.1.7';
```
Comment (tr1b0t) [inline src/Tribe/Main.php:38]: :warning: Equals sign not aligned correctly; expected 1 space but found 13 spaces

<sub>hash: 1eb125f5345e78d2763bd5b0802a3232</sub>

**Comment by lucatume** [comment]
I'm merging this one, I will deactivate the Travis CI integration after this one is merged.

**Pull request merged**